code/obj/item/grenades.dm

```diff
@@ -1402,7 +1402,7 @@ ADMIN_INTERACT_PROCS(/obj/item/gimmickbomb, proc/arm, proc/detonate)
 						if (src)
 							src.boom()
 							if (target)
-								if (istype(target, /obj/machinery))
+								if (istype(target, /obj/machinery) && !istype(target, /obj/machinery/atmospherics/binary/nuclear_reactor))
```

Comment:
Perhaps it would be better to make it specifically check for doors, if that code block is meant for busting doors 🤔

Reply:
Maybe, but people might be used to them removing stuff like computers, manufacturers, the AI law rack and the PTL. For the first 2, it's probably a good thing that they get nuked, for the law rack and PTL, I dunno, it'd be a gameplay change.

Thoughts on what's preferred between a blacklist or whitelist and it's content?